tests/pdctl/unsafe/unsafe_operation_test.go

```diff
@@ -49,6 +49,9 @@ func (s *unsafeOperationTestSuite) TestRemoveFailedStores(c *C) {
 	args := []string{"-u", pdAddr, "unsafe", "remove-failed-stores", "1,2,3"}
 	_, err = pdctl.ExecuteCommand(cmd, args...)
 	c.Assert(err, IsNil)
+	args = []string{"-u", pdAddr, "unsafe", "remove-failed-stores", "1,2,3", "3600"}
```

Comment:
we may need to add more options like `--do-recreate --allow-learner --timeout 3600`, can we support specifying it with a option name?

Reply:
Done.

Reply:
Changed it to a "if" condition, since we currently only have 1 parameter and golang linter does not allow me use "switch".